R/environment.R

```diff
@@ -256,10 +256,23 @@ generate_docker_file <- function(custom_docker_image = NULL,
 
   if (!is.null(cran_packages)) {
     for (package in cran_packages) {
+
+      # parse to split package name from version, if version provided
```

Comment:
I think we should add R Section in R SDK to give proper structure to the packages. Also, we should add "version" parameter in the RCranPackage section.

For now, we can go ahead with this PR but let's create a work item to add rsection in the sdk.
@mx-iao what do you suggest?

Reply:
Sounds good, let's make this patch for now.